.goreleaser.yaml

```diff
@@ -0,0 +1,86 @@
+project_name: flasher
+before:
+  hooks:
+    - go mod tidy
+
+builds:
+  - id: go
+    env:
+      - CGO_ENABLED=0
+    goos:
+      - linux
+      - darwin
+    ldflags:
+      - -X "github.com/metal-toolbox/flasher/internal/version.AppVersion={{ .Version }}"
```

Comment:
we don't have a version command in mctl yet, maybe we should add that in so users can verify the version thier running?

Reply:
I added a version command now.